csharp/src/Ice/Proxy.cs

```diff
@@ -526,6 +543,33 @@ async Task<IncomingResponseFrame> InvokeAsync()
                     observer?.Detach();
                 }
             }
+
+            void TraceRetry(string message, int retryCount, RetryPolicy policy, Exception? exception = null)
+            {
+                var sb = new StringBuilder();
+                sb.Append(message);
+                sb.Append("\nproxy = ");
+                sb.Append(proxy);
+                sb.Append("\noperation = ");
+                sb.Append(request.Operation);
+                sb.Append("\nretry count = ");
+                sb.Append(retryCount);
```

Comment:
This shows 0/5 for the first retry and 4/5 for the last which is a bit odd to me, I think it should be 1/5 and 5/5, keep in my that RetryMaxAttempts include the first invocation